Positioning2/positioning.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="parent"><p><b>PARENT</b></p>
```

Comment:
As transforma textul tot intr-un h2
```suggestion
    <div class="parent">
      <h2>PARENT<h2>
```